apistructs/issue.go

```diff
@@ -686,6 +686,8 @@ type IssueListRequest struct {
 	IdentityInfo
 	// 用来区分是通过ui还是bundle创建的
 	External bool `json:"-"`
+	// simple
+	SimpleResult bool `json:"simpleResult"`
```

Comment:
No one knows how many fields will be returned in Simple-Mode.
Maybe `OnlyID` is better.